app/src/lib/stores/app-store.ts

```diff
@@ -1731,6 +1739,8 @@ export class AppStore extends TypedBaseStore<IAppState> {
       }
     })
 
+    this.hasShownCherryPickIntro = getBoolean(hasShownCherryPickIntroKey, true)
```

Comment:
Shouldn't the default be `false`? 🤔 
```suggestion
    this.hasShownCherryPickIntro = getBoolean(hasShownCherryPickIntroKey, false)
```

Reply:
Hmm.. Yes.. but why did this work? 😄

Reply:
🤦  It doesn't.. I just already had the key...

Reply:
:magic: 🤣